tapiriik/services/Coros/coros.py

```diff
@@ -306,5 +306,4 @@ def ExternalIDsForPartialSyncTrigger(self, req):
             if 'openId' in item:
                 external_user_ids.append(item['openId'])
                 logging.info("[WEBHOOK] COROS CALLBACK for user id %s, and activity id %s" % (item['openId'], item['labelId']))
-                return external_user_ids
-        return []
+        return external_user_ids
```

Comment:
on se retrouve avec des doublons dans la liste des ids dans ce cas non ?

Reply:
yep, l'appelant doit aussi être modifier non ? 
Le mieux c'est de rajouter un test avec une notif et deux users différents

Reply:
Dédoublage présent en dehors de cette méthode.

Pour résumer cette liste on l'utilise juste après pour faire une requête mongo en utilisant l'opérateur [$in](https://www.mongodb.com/docs/manual/reference/operator/query/in/#-in)

Le code est le suivant :
![image](https://user-images.githubusercontent.com/74352754/189850175-ecf4eed1-ae4e-46ab-9742-f7623a9b3af3.png)

Reply:
ca sert à quoi de sortir une liste avec des doublons ?
Si on a 100 activités pour une personne on obtient une liste de 100 ids identiques ?
Ca n'a pas de sens - à moins que ca ne soit requis pour un algo présent dans le hub ailleurs
Sinon il faut un set

Reply:
Je suis d'accord mais dans ce cas il faudrait aussi modifier le code FITBIT et de GARMIN qui permettent aussi les doublons